architecture/infrastructure_components/kubernetes_infrastructure.adoc

```diff
@@ -104,6 +104,14 @@ source their configuration from the *_/etc/origin/master/master-config.yaml_*
 file, as described in the
 xref:../../install_config/master_node_configuration.adoc#install-config-master-node-configuration[Master and Node Configuration] topic.
 
+[discrete]
+[[control-plane-startup-sequence]]
+==== Startup Sequence
+
+This section covers the startup sequence and the different pieces involved to aid with understanding and troubleshooting.
+
+Hyperkube is a binary that contains all of Kubernetes (kube-apiserver, controller-manager, scheduler, proxy, and kubelet). On startup, the kubelet creates the kubepods.slice. Next, the kublet creates the QoS-level slices burstable.slice and best-effort.slice inside the kubepods.slice. When a pod starts, the kubelet creats a pod-level slice with the format `pod<UUID-of-od>.slice` and passes that path to the runtime on the other side of the CRI. Docker or CRI-O then creates the container-level slices inside the pod-level slice.
```

Comment:
Please review the [markup reference](https://github.com/openshift/openshift-docs/blob/master-3/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference) for the 3.x collection and update the syntax highlighting for the package and program names.

Reply:
What is `UUID-of-od`? Do you mean pod?

Reply:
I'd define CRI because it's not defined in this context.